include/zephyr/acpi/acpi.h

```diff
@@ -19,6 +19,22 @@ struct acpi_dev {
 	ACPI_DEVICE_INFO *dev_info;
 };
 
+union acpi_dmar_id {
+	struct {
+		uint16_t function: 3;
+		uint16_t device: 5;
+		uint16_t bus: 8;
+	} bits;
+
+	uint16_t raw;
+};
+
+struct acpi_mcfg {
+	struct acpi_table_header Header;
```

Comment:
why the capital letter ? 

Unless we use an externally declared structures/variable/etc... (so from hal, lib etc...), we do not use came case in Zephyr. So here, just use small letter.